setup.py

```diff
@@ -1,150 +1,5 @@
-#! /usr/bin/env python
-"""Ansible-lint distribution package setuptools installer."""
-
+#!/usr/bin/env python
```

Comment:
Plz, don't remove the space after `#!`, it's there for readability.
```suggestion
#! /usr/bin/env python
```